kusd/validator/states.go

```diff
@@ -69,6 +69,16 @@ func (val *Validator) notLoggedInState() stateFn {
 			}
 		}
 	} else {
+		// sanity check
+		isVoter, err := val.network.IsVoter(&bind.CallOpts{}, val.account.Address)
```

Comment:
this is the same code as above, if we exit on err always  (log.Crit?) maybe is worth moving this into a assertIsVoter/assertGetIsVoter (or similar) method helper returning only a bool as assert implies failure/panic. 

this methos is long already, is the block above a makeDeposit()?

I feel like i've done something today :)